.travis.yml

```diff
@@ -16,6 +16,7 @@ addons:
     - pandoc
 before_install:
   - pip install invoke
+  - pip install codecov
```

Comment:
Could you instead put codecov in the dev-requires in flit.ini and remove coveralls?